dbt/adapters/fabricspark/shortcuts.py

```diff
@@ -153,8 +153,32 @@ def check_exists(self, shortcut: Shortcut):
         # check if the error is ItemNotFound
         if response.status_code == 404:
             return False
-        response.raise_for_status()
+        response.raise_for_status() # raise an exception if there are any other errors
```

Comment:
should we specifically look for success status before doing this? I'm not sure what the behavior is in case of other 4xx errors.

Reply:
we can. [here's](https://requests.readthedocs.io/en/latest/_modules/requests/models/#Response.raise_for_status) the source code for `raise_for_status` -- it just raises an HTTPError if the status code of the response is between 400-600.

Reply:
great, this is good then, thanks!